data/json/vehicle_parts.json

```diff
@@ -3659,7 +3659,7 @@
         "range" : 12,
         "fuel_type" : "50",
         "item" : "m2browning",
-        "difficulty" : 6,
+        "difficulty" : 4,
```

Comment:
The M2's known for recoil.  I'd say mounting one someplace that most likely isn't built to take the punishment would be 5, maybe leave it at 6.